projects/fractally/apps/webapp/src/app/video/components/in-call.tsx

```diff
@@ -135,22 +141,22 @@ export const InCall = ({ joinDetails, onLeave }) => {
                 <nav className="flex-col space-y-2 mt-3">
                     <section>
                         <DeviceSelect
-                            onChange={(id) => {
-                                room.updateCamera({ deviceId: id });
+                            onChange={async (id) => {
+                                await room.updateCamera({ deviceId: id });
                             }}
                             deviceName="Camera"
                             devices={cameras}
                         />
                         <DeviceSelect
-                            onChange={(id) => {
-                                room.updateMicrophone({ deviceId: id });
+                            onChange={async (id) => {
+                                await room.updateMicrophone({ deviceId: id });
                             }}
                             deviceName="Microphone"
                             devices={microphones}
                         />
                         <DeviceSelect
-                            onChange={(id) => {
-                                room.updateSpeaker({ deviceId: id });
+                            onChange={async (id) => {
+                                await room.updateSpeaker({ deviceId: id });
```

Comment:
Correct me if I'm wrong, but adding `async...await` syntax in these four instances in this file is superfluous. It's not necessary. These async methods (`room.updateSpeaker(...)`, `room.updateCamera(...)`, etc.) return Promises, like any async function. The same could be achieved by adding the `await` keyword to `<DeviceSelect />` on line 21 where it calls this `onChange()` prop. But even there, it's unnecessary (and actually achieves nothing) because these `onChange` props values return `void`. Nothing is waiting for the return value from these promises. Same for `room.leave()` below this.

Did this change definitely fix a bug for you. Or am I thinking about this incorrectly?

This extra syntax certainly doesn't hurt anything. I'm just asking more for my own curiosity.